packages/app-desktop/gui/JoplinCloudLoginScreen.tsx

```diff
@@ -71,10 +71,19 @@ const JoplinCloudScreenComponent = (props: Props) => {
 	};
 
 	useEffect(() => {
+		const removePreviousSyncReportMessage = () => {
+			if (state.active !== 'COMPLETED') {
+				props.dispatch({
+					type: 'SYNC_REPORT_UPDATE',
+					report: {},
+				});
+			}
+		};
 		return () => {
 			clearInterval(intervalIdentifier);
+			removePreviousSyncReportMessage();
 		};
-	}, [intervalIdentifier]);
+	}, [intervalIdentifier, state, props.dispatch]);
```

Comment:
We should only pass what's needed for the effect, so only state.active here

Reply:
Also I don't really understand why this effect does and why it's modifying global state?